nzedb/Backfill.php

```diff
@@ -112,21 +115,21 @@ public function __construct(array $options = [])
 			}
 		}
 
-		$this->_compressedHeaders = (Settings::value('..compressedheaders') == 1 ? true : false);
-		$this->_safeBackFillDate = (Settings::value('..safebackfilldate') != '') ?
-			(string)Settings::value('..safebackfilldate') : '2008-08-14';
-		$this->_safePartRepair = (Settings::value('..safepartrepair') == 1 ? 'update' : 'backfill');
-		$this->_disableBackfillGroup = (Settings::value('..disablebackfillgroup') == 1 ? true : false);
+		$this->_compressedHeaders = (int) Settings::value('..compressedheaders') === 1;
```

Comment:
Why cast to integer? The original specifically made the variable a boolean, a more strict type for easier checking.

Reply:
Forget that, I was misreading it.

Reply:
I think our code-style is to not have a space between a cast and the variable.